crates/cargo-platform/src/check_cfg.rs

```diff
@@ -0,0 +1,71 @@
+//! check-cfg
+
+use std::collections::{HashMap, HashSet};
+
+/// Check Config (aka `--print=check-cfg` representation)
+#[derive(Debug, Default, Clone)]
+pub struct CheckCfg {
```

Comment:
So this isn't actually a `check-cfg` parser but a `print-check-cfg` parser, right?  Seems like the name should convey that.

Reply:
The `CheckCfg` struct is representing the internal of check-cfg, it doesn't really have anything to do with how it's parsed. (The compiler is using the same representation [internaly](https://doc.rust-lang.org/stable/nightly-rustc/rustc_session/config/struct.CheckCfg.html))

It *could* be used to parse the `--check-cfg` without an issue.

Instead of renaming the struct what about renaming the parsing method instead?
`CheckCfg::process_line` -> `CheckCfg::parse_print_check_cfg_line`?

Reply:
> Instead of renaming the struct what about renaming the parsing method instead?

That might work

Reply:
Renamed to `CheckCfg::parse_print_check_cfg_line`.